sdks/python/apache_beam/yaml/examples/test_read_mysql_write_postgres.yaml

```
@@ -0,0 +1,48 @@
+# coding=utf-8
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This examples reads from a public file stored on Google Cloud. This
+# requires authenticating with Google Cloud, or setting the file in
+#`ReadFromText` to a local file.
+#
+# To set up Application Default Credentials,
+# see https://cloud.google.com/docs/authentication/external/set-up-adc for more
+# information
+#
+# The following example reads data from a Mysql database
+# then load the results back to a Postgre database
+pipeline:
+  transforms:
+    - type: ReadFromJdbc
+      name: ReadFromMysql
+      config:
+        driver_class_name: 'com.mysql.cj.jdbc.Driver'
+        jdbc_url: 'jdbc:mysql://database_ip:3306/database_name'
+        username: 'username'
+        password: 'password'
+        read_query: 'select * from table;'
+        driver_jars: 'mysql-connector-j-9.0.0.jar'
```

You can use the `ReadFromMySql` transform to be less verbose and not require staging a driver JAR
```suggestion
    - type: ReadFromMySql
      config:
        url: 'jdbc:mysql://database_ip:3306/database_name'
        username: 'username'
        password: 'password'
        query: 'select * from table;'
```